python/caffe/draw.py

```diff
@@ -142,9 +142,10 @@ def get_pydot_graph(caffe_net, rankdir, label_edges=True):
     -------
     pydot graph object
     """
-    pydot_graph = pydot.Dot(caffe_net.name,
-                            graph_type='digraph',
-                            rankdir=rankdir)
+    dot_args = {'graph_type': 'digraph', 'rankdir': rankdir}
+    if caffe_net.name:
+        dot_args['graph_name'] = caffe_net.name
+    pydot_graph = pydot.Dot(**dot_args)
```

Comment:
I agree; this is more readable than the proposed change.
(typo: `caffe.net` --> `caffe_net`)